tests/aws/services/cloudformation/resource_providers/scheduler/test_scheduler.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from localstack.testing.pytest import markers
+
+
+@markers.aws.validated
+@markers.snapshot.skip_snapshot_verify(paths=["$..DriftInformation", "$..Metadata"])
+def test_schedule_and_group(deploy_cfn_template, aws_client, snapshot):
+    stack = deploy_cfn_template(
+        template_path=os.path.join(os.path.dirname(__file__), "templates/schedule.yml")
+    )
+
+    snapshot.add_transformer(
+        snapshot.transform.key_value("PhysicalResourceId", "physical_resource_id")
+    )
+    snapshot.add_transformer(snapshot.transform.cloudformation_api())
+    schedule = aws_client.cloudformation.describe_stack_resource(
+        StackName=stack.stack_name, LogicalResourceId="MySchedule"
+    )["StackResourceDetail"]
+    snapshot.match("Schedule", schedule)
+
+    group = aws_client.cloudformation.describe_stack_resource(
+        StackName=stack.stack_name, LogicalResourceId="MyScheduleGroup"
+    )["StackResourceDetail"]
+    snapshot.match("Group", group)
```

Comment:
would be great if we could add some specific API describe calls as well, i.e. get_schedule / get_schedule_group